packages/react/src/app/index.css

```diff
@@ -0,0 +1,92 @@
+:root {
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  line-height: 1.5;
+  font-weight: 400;
+
+  color-scheme: light;
+
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  --color-primary: #4a90e2; /* Tailwind's blue-500 */
+  --color-primary-dark: #357ebd; /* Tailwind's blue-600 */
+  --color-primary-darker: #276bb0; /* Tailwind's blue-700 */
+  --color-white: #ffffff; /* Tailwind's white */
+  --color-gray-100: #f7fafc; /* Tailwind's gray-100 */
```

Comment:
Similar comment to https://github.com/zendesk/app_scaffolds/pull/16/files#r1650215828. Rather than colors, any reason we can't pull in Zendesk garden colors?